app/controllers/subsidy/measure-offer.js

```diff
@@ -1,7 +1,84 @@
 import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import moment from 'moment';
+import { inject as service } from '@ember/service';
 
 export default class SubsidyMeasureOfferController extends Controller {
-  sort = '-modified';
-  page = 0;
-  size = 20;
+  @service router;
+  @service store;
+
+  @tracked selected = null;
+  @tracked options = [];
+  @tracked startDate;
+  @tracked endDate;
+
+  async setup() {
+    await this.loadOptions();
+  }
```

Comment:
```suggestion
  async setup() {
    this.selected = null; //reset the previous selected value
    await this.loadOptions();
  }
```
This will erase the previous series, otherwise it shows up a few seconds while the new series is loading which gets confusing

Reply:
yes saw it as well, was already working on that, used AuLoader instead to avoid confusion

Reply:
Ahh perfect